ARCHITECTURE.md

```diff
@@ -6,6 +6,7 @@ For further information, see also:
 
 - [Kedro-Viz contributing documentation](CONTRIBUTING.md), which covers how to start development on the project
 - [Kedro-Viz style guide](STYLE_GUIDE.md), which walks through our standards and recommended best practices for our codebase
+- [Kedro-Viz Architecture Diagram](https://miro.com/app/board/uXjVKhNg1RE=/?moveToWidget=3458764606468376036&cot=10), to see a high level overview of both back-end and front-end and how they are connected.
```

Comment:
Is the `img/frontend-architecture.png` the only screenshot available? Given that the Miro board won't be accessible by the public, maybe we should try to capture the full diagram for archival purposes.

Reply:
agree, can we have a backend architecture section as well and do a short writeup and have the image ? thanks

Reply:
Done!